lib/utils.js

```diff
@@ -54,6 +54,15 @@ export function isResSent (res) {
 }
 
 export async function loadGetInitialProps (Component, ctx) {
+  if (
+    process.env.NODE_ENV !== 'production' &&
```

Comment:
This should actually be a completely separate statement:

```
if(process.env.NODE_ENV !== 'production') {
  if(Component.prototype && Component.prototype.getInitialProps) {
    // etc
  }
}
```

This makes sure uglify removes it in production.

Reply:
Good point. Thanks!